axes_nohw.py

```diff
@@ -0,0 +1,204 @@
+import logging
```

Comment:
I see how an emulation for testing is super useful because we don't always have the raspi and other hw available. Is duplicating the files (axes.py and led.py) the best way to do this? For a quick fix, it is fine. I don't know if there is an easy and elegant solution if it something we want to keep...

Reply:
I think the Led case is solved quite nicely:
`try:  
    from led import Leds
except:
    from ledemul import Leds`
I think the axes could be solved similarly